packet_flight/js/packet-flight.js

```diff
@@ -213,6 +210,15 @@ function Packet(sendr, recvr, delay, bytes, type, flight_time) {
   this.type = type;
 }
 
+function blankTheCanvas(paper) {
+  paper.setStart();
+  paper.circle(16, 16, 15).attr({ fill: 'white' }).data('bound', true);
+  paper.circle(16, 16, 10);
+  paper.path('M16,6L16,9M21,7L19.5,10M25,11L22,12.5M26,16L23,16M25,21L22,19.5M21,25L19.5,22M16,26L16,23M11,25L12.5,22M7,21L10,19.5M6,16L9,16M7,11L10,12.5M11,7L12.5,10M18,9L16,16L20,16');
```

Comment:
Since I'm not familiar with the `raphael` library, could you explain what is exactly going on here?